docs/UserGuide.md

```diff
@@ -41,27 +53,25 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ## Features
 
-<div markdown="block" class="alert alert-info">
+<details>
+  <summary style="font-weight:600;font-size:20px;">Notes about the command format:</summary>
 
-**:information_source: Notes about the command format:**<br>
-
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
+  + Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
   e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  + Items in square brackets are optional.<br>
+    e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```

Comment:
this example is using t/ which is not optional in our system. same for below

Reply:
Didn't update the content in here, only updated the structure to get a collapsible notes section for the features!
- content to be updated by respective authors of the noted features